backend/api/serializers.py

```diff
@@ -190,13 +190,26 @@ class Meta:
 
 class UserSerializer(serializers.ModelSerializer):
     organization = OrganizationSerializer(read_only=True)
+    organization_balance = serializers.SerializerMethodField()
 
     class Meta:
         model = User
         fields = (
             'id', 'first_name', 'last_name', 'email', 'authorization_id',
             'authorization_guid', 'authorization_directory', 'display_name',
-            'organization')
+            'organization', 'organization_balance')
+
+    def get_organization_balance(self, obj):
+        try:
```

Comment:
Maybe a bit pedantic, but if this is not an exceptional circumstance, one shouldn't be handling it as an exception, but rather as a check. I would expect exception handling to be involved if something is actually wrong, and it be logged etc.